kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/OfferUtils.java

```diff
@@ -3,6 +3,8 @@
 import org.apache.mesos.Protos.Label;
 import org.apache.mesos.Protos.TaskInfo;
 
+import com.mesosphere.dcos.kafka.config.JmxConfig;
+
 public class OfferUtils {
```

Comment:
`OfferUtils` doesn't seem like the right location for this.  Maybe make a class over in the `com.mesosphere.dcos.kafka.config` package.